final_task/rss_reader/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3.7'
+
+services:
+
+  flask:
+    build: .
+    restart: always
+    container_name: flask_app
+    ports:
+      - 5000:5000
+    volumes:
+      - .:/rss_reader_app
+    env_file: 
+      - ./.env.dev
+    depends_on: 
+      - localhost
+    links: 
+      - localhost
+
+  localhost:
```

Comment:
Why you database service is called `localhost`?

Reply:
after testing forgot to change